packages/react-native-windows-init/src/Cli.ts

```diff
@@ -13,10 +13,56 @@ import * as validUrl from 'valid-url';
 import * as prompts from 'prompts';
 import * as findUp from 'find-up';
 import * as chalk from 'chalk';
-
+import * as path from 'path';
 // @ts-ignore
 import * as Registry from 'npm-registry';
 
+import * as appInsights from 'applicationinsights';
+import {randomBytes} from 'crypto';
+
+/**
+ * Important:
+ * Do not use process.exit() in this script as it will prevent telemetry from being sent.
+ * See https://github.com/microsoft/ApplicationInsights-node.js/issues/580
+ */
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
+const telClient = appInsights.defaultClient;
+telClient.commonProperties.sessionId = randomBytes(16).toString('hex');
+if (process.env.RNW_CLI_TEST) {
+  telClient.commonProperties.isTest = process.env.RNW_CLI_TEST;
+}
+
+function sanitizeStackTrace(envelope: any /*context: any*/): boolean {
```

Comment:
Again, duplication. If not practical to share, please add a comment like
```ts
// Code-Sync: .....\bla.ts
// Code-Sync: .....\foo.ts
```
Creating full cross links in case someone fixes a bug in one place the know to apply it to the other locations